plugins/modules/inventory_intent.py

```diff
@@ -2635,41 +2708,43 @@ def get_diff_deleted(self, config):
                     params={"id": udf_id},
                 )
                 if response and isinstance(response, dict):
+                    self.log("Received API response from 'delete_user_defined_field': {0}".format(str(response)), "DEBUG")
                     task_id = response.get('response').get('taskId')
 
                     while True:
                         execution_details = self.get_task_details(task_id)
 
                         if 'success' in execution_details.get("progress"):
-                            self.msg = "Global UDF - {0} Deleted Successfully from Cisco DNA Center".format(field_name)
                             self.status = "success"
+                            self.msg = "Global UDF '{0}' deleted successfully from Cisco Catalyst Center".format(field_name)
+                            self.log(self.msg, "INFO")
                             self.result['changed'] = True
                             self.result['response'] = execution_details
                             break
                         elif execution_details.get("isError"):
                             self.status = "failed"
                             failure_reason = execution_details.get("failureReason")
                             if failure_reason:
-                                self.msg = "Global UDF Deletion get failed because of {0}".format(failure_reason)
+                                self.msg = "Global UDF deletion get failed because of {0}".format(failure_reason)
                             else:
-                                self.msg = "Global UDF Deletion get failed."
-                            self.log(self.msg)
+                                self.msg = "Global UDF deletion get failed."
```

Comment:
self.msg = "Failed to delete Global User Defined Fields(UDFs) due to: {0}".format(failure_reason)

Reply:
added.